README.md

```diff
@@ -372,6 +372,47 @@ $ npm run build
 
 ## Run Hyperledger Explorer
 
+### Bootup Mode
+
+The Bootup Mode feature allows you to specify how many blocks should be loaded when starting the Hyperledger Explorer. You can choose from two modes:
+
+-   **ALL**: Load all available blocks.
+-   **CUSTOM**: Load a specific number of blocks as configured in the `config.json` file.
+
+**Note:** If you provide any other mode, it will Bootup Hyperledger explorer with latest block.
+
+To set the Bootup Mode, update the `app/platform/fabric/config.json` file in your project with the desired mode.
+
+-   **ALL**
+      ```json
+        {
+            "network-configs": {
+                "test-network": {
+                    "name": "Test Network",
+                    "profile": "./connection-profile/test-network.json",
+                    "enableAuthentication": false,
+                    "bootMode": "ALL",
```

Comment:
Please add the "noOfBlocks" as "0" in ALL mode as well

Reply:
Added, noOfBlocks parameter as "0" below "ALL" mode also.